Cargo.toml

```
@@ -5,6 +5,7 @@ members = [
     "extensions/scarb-cairo-language-server",
     "extensions/scarb-cairo-run",
     "extensions/scarb-cairo-test",
+    "extensions/scarb-test-collector",
```

```suggestion
    "extensions/scarb-snforge-test-collector",
```

`test-collector` by itself was pretty ambiguous IMO